Makefile

```diff
@@ -229,8 +229,10 @@ images: $(IMAGES_BUILD)
 
 ### Typechecking ###
 # grab the bin from https://github.com/facebook/flow/releases
-flow: build/js/all.js
-	grep -Ev '/\*:: *(ex|im)port ' $< | flow check-contents
+flow: $(JSFILES)
+	for file in $(JSFILES); do \
```

Comment:
Instead of `$(JSFILES)` this should probably be `$<`?

Reply:
Tried that. Something broke. Will try again.